story.html

```diff
@@ -15,15 +16,25 @@ <h1>Mad Libs</h1>
       <section>
         <h2>Birthday Party Fun 🎂</h2>
         <p>
-          Yesterday I went to {personsName}'s birthday party. I got him/her a
-          {noun1}. The party was {adjective}. We started by playing {activity}
-          and then there was a {verb} party. Lots of my friends were there but I
-          mostly hung out with {personsName2}. We talked about {noun2}, and how
-          our friend {personsName3} is a {noun3}. During cake everyone
-          {pastTenseVerb} and sang {songName}. I had a/an {adjective2} time at
-          the party and enjoyed celebrating {personsName}. He/she is such a
-          {adjective3} friend.
+          Yesterday I went to <span>{personsName}</span>'s birthday party. I got
+          him/her a <span>{noun1}</span>. The party was
+          <span>{adjective}</span>. We started by playing
+          <span>{activity}</span> and then there was a
+          <span>{verb}</span> party. Lots of my friends were there but I mostly
+          hung out with <span>{personsName2}</span>. We talked about
+          <span>{noun2}</span>, and how our friend
+          <span>{personsName3}</span> is a <span>{noun3}</span>. During cake
+          everyone <span></span>{pastTenseVerb} and sang
```

Comment:
this one also seems like `{pastTenseVerb}` should be inside of the `<span>` element